apps/developer-hub/src/components/Pages/PriceFeedsLandingPage/index.module.scss

```diff
@@ -0,0 +1,120 @@
+@use "@pythnetwork/component-library/theme";
+
+.container {
+  max-width: 80rem; /* ~ max-w-7xl */
+  margin-left: auto;
+  margin-right: auto;
+  padding: theme.spacing(8) theme.spacing(4) theme.spacing(8) theme.spacing(4); /* px-4 */ /* py-8 */
+}
+
+@media (width >= 640px) {
+  .container {
+    padding-left: theme.spacing(6); /* sm:px-6 */
+    padding-right: theme.spacing(6);
+  }
+}
+
+@media (width >= 1024px) {
+  .container {
+    padding-left: theme.spacing(8); /* lg:px-8 */
+    padding-right: theme.spacing(8);
+  }
+}
```

Comment:
You should be nesting these inside `.container`, and you should use `theme.breakpoint` for these, e.g. this:

```scss
.container {
  max-width: ...;

  @include theme.breakpoint("sm") {
    ...
  }
}
```